### ISSUE TITLE
“Home” link in “Edit page” has no description

### ISSUE THREAD

**Issue by lkiesow** [elan-ev/tobira#646]
The “home” link on the “Edit page” view has no description. If you are using a screen reader, you just get the information that it's a link. That is not helpful.

![Screenshot from 2023-01-05 19-28-44](https://user-images.githubusercontent.com/1008395/210855089-4a300294-84e4-4d2b-9fef-47d1b44e0a55.png)

**Comment by owi92** [comment]
After reading around a little (I couldn't reproduce this with VoiceOver for mac in firefox), the culprit here appears to be the usage of the html `title` attribute, which seems to be quite unreliable for accessibility concers as some (or even most?) screen readers do not use it.
As the `title` attribute is used in a lot of places, I guess this is a larger issue (but still not hard to fix, I think).

It could get fixed by replacing it with the `aria-label` attribute, but I would probably wait for #636 as this already replaces the `title` in a couple of places with a proper tooltip element, which hopefully also works and could or should be user over the `aria-title`.
